src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/TransactionTypesSection/GnosisNFTTransfer.css

```diff
@@ -0,0 +1,93 @@
+@value width from '../GnosisControlSafeForm.css';
+
+.nftPicker,
+.recipientPicker {
+  margin: 20px 0;
+}
+
+.nftPicker > div,
+.recipientPicker > div {
+  padding-left: 0px;
+}
+
+.nftPicker input,
+.recipientPicker input {
+  width: width;
```

Comment:
@danbr This fixed width is responsable for the formatting issue I described. Changing it to 100% solves it on my machine.

Reply:
@willm30 do other instances of the picker display correctly for you? I can change it to 100% but just surprised that you dont have the same problem else where, with other transfer types.

Reply:
@danbr Yes, the other ones format correctly but they are also using `width: 100%` (taken from the SingleUserPicker component itself)